README.md

```diff
@@ -2,7 +2,7 @@
 
 A 12 column responsive grid system for laying out documents, templates and modules.
 
-> Living off the grid and being kind of an outlaw brings a dangerous reality.  
+> Living off the grid and being kind of an outlaw brings a dangerous reality.
```

Comment:
I believe that's just trailing whitespace? It's a single line change, shouldn't be a line break. I also don't remember touching that line, must've been an editor plugin.